# The joint state controller handles publishing transforms for any moving joints
floribot:
   joint_state_controller:
      type: "joint_state_controller/JointStateController"
      publish_rate: 50

   joint_fl_controller:
      type: velocity_controllers/JointVelocityController
      joint: joint_frontLeft
      pid: {p: 100.0, i: 0.01, d: 10.0}
      
   joint_fr_controller:
      type: velocity_controllers/JointVelocityController
      joint: joint_frontRight
      pid: {p: 100.0, i: 0.01, d: 10.0}
      
   joint_rl_controller:
      type: velocity_controllers/JointVelocityController
      joint: joint_rearLeft
      pid: {p: 100.0, i: 0.01, d: 10.0}
      
   joint_rr_controller:
      type: velocity_controllers/JointVelocityController
      joint: joint_rearRight
      pid: {p: 100.0, i: 0.01, d: 10.0}

    gazebo_ros_control:
      pid_gains:
       joint_frontLeft: {p: 1.0, i: 0.0, d: 0.0}
       joint_frontRight: {p: 1.0, i: 0.0, d: 0.0}
       joint_rearLeft: {p: 1.0, i: 0.0, d: 0.0}
       joint_rearRight: {p: 1.0, i: 0.0, d: 0.0}


